src/transaction/locator_sr.c

```diff
@@ -12606,7 +12606,17 @@ locator_is_exist_class_name_entry (THREAD_ENTRY * thread_p, LOCATOR_CLASSNAME_EN
       assert (entry->e_tran_index == NULL_TRAN_INDEX);
 
       assert (!OID_ISNULL (&entry->e_current.oid));
-      assert (heap_does_exist (thread_p, oid_Root_class_oid, &entry->e_current.oid));
+      /* TODO:
```

Comment:
@joohok 
Please allow me to ask: how do we plan to track these 'TODO' scaterred around the code?

I have to admit that I also added a bunch of them, but they are mostly in new code.

Is it reasonable to ask to track isolated TODO's in legacy code - such as this one by explicit Jira issues? At least to have them all in Jira and someone/sometime will see them there :)

Thanks.

Reply:
This PR is a temporary fix as we couldn't determine the design direction for LETS-814.
LETS-814 will soon address this TODO. However, the TODO is not explicitly mentioned in the LETS-814 issue. 
I should document it explicitly in LETS-814 for now.

I agree with your suggestion to create and track an issue for any TODOs that arise. Especially if TODOs in legacy code are not tracked, it can pose risks

Thank you